AddIamRoleToNeptuneStack Timeout

I'm trying to deploy the Cloudformation script using the AWS Cloudformation dashboard found at the links in this repo.  My AWS account is in US East 2, and I'm clicking the corresponding link (`https://us-east-2.console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/create/review?templateURL=https://s3.amazonaws.com/aws-admartech-samples-us-east-2/identity-resolution/templates/identity-resolution.yml&stackName=Identity-Graph-Sample`).  

I've tried several times, but it always fails and rolls back with a timeout error at the `AddIamRoleToNeptuneStack` step.  When I dig into the logs, I see the following error:
```
[ERROR] DBClusterRoleAlreadyExistsFault: An error occurred (DBClusterRoleAlreadyExists) when calling the AddRoleToDBCluster operation: Role ARN arn:aws:iam::087756882422:role/Identity-Graph-Sample-Neptun-NeptuneLoadFromS3Role-Z2KDEBED48SM is already associated with DB Cluster: neptunedbcluster-a1epw1ieprh8. Verify your role ARN and try again.
Traceback (most recent call last):
  File "/var/task/index.py", line 18, in handler
    addIamRole(dbClusterId, iamRoleArn)
  File "/var/task/index.py", line 37, in addIamRole
    RoleArn=iamRoleArn
  File "/var/runtime/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/var/runtime/botocore/client.py", line 676, in _make_api_call
    raise error_class(parsed_response, operation_name)
```

I just started using Terraform, and don't have any experience with Cloudformation.  I'm also pretty green when it comes to IAM, VPC's etc.   However, when I check our IAM roles, I see no role with `Neptune` or `Graph` in the name, so I assume this conflicting role only exists temporarily while Cloudformation is running, or it exists in someone else's account?  I believe I have permissions to see all of the roles in our AWS account.  Thanks for any help!

--- a/identity-resolution/templates/identity-resolution.yml
+++ b/identity-resolution/templates/identity-resolution.yml
@@ -5,7 +5,7 @@ Mappings:
     us-west-2:
       bucket: aws-admartech-samples-us-west-2
     us-east-1:
-      bucket: aws-admartech-samples
+      bucket: aws-admartech-samples-us-east-1
     us-east-2:
       bucket: aws-admartech-samples-us-east-2
     eu-west-1:
@@ -13,7 +13,7 @@ Mappings:
 
   Constants:
     S3Keys:
-      neptuneNotebooks: /identity-resolution/notebooks|identity-resolution/*
+      neptuneNotebooks: /identity-resolution/notebooks|identity-graph/*
       irdata: /identity-resolution/data/
       bulkLoadStack: /identity-resolution/templates/bulk-load-stack.yaml
 
@@ -78,22 +78,6 @@ Resources:
                                               - Constants
                                               - S3Keys
                                               - neptuneNotebooks
-      TimeoutInMinutes: '60'
-
-# ---------- CONNECTING IAM ROLE TO NEPTUNE CLUSTER ----------
-  AddIamRoleToNeptuneStack:
-    Type: AWS::CloudFormation::Stack
-    Properties:
-      TemplateURL: https://s3.amazonaws.com/aws-neptune-customer-samples/neptune-sagemaker/cloudformation-templates/common/add-iam-role-to-neptune.json
-      Parameters:
-        DBClusterId:
-          Fn::GetAtt:
-          - NeptuneBaseStack
-          - Outputs.DBClusterId
-        NeptuneLoadFromS3IAMRoleArn:
-          Fn::GetAtt:
-          - NeptuneBaseStack
-          - Outputs.NeptuneLoadFromS3IAMRoleArn
       TimeoutInMinutes: '60'
 
 # --------- LOAD DATA INTO NEPTUNE ---------

